mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -3321,6 +3366,91 @@ void IfOp::print(OpAsmPrinter &p) {
   p.printOptionalAttrDict((*this)->getAttrs());
 }
 
+LogicalResult IfOp::verify() {
+  if (getThenGraph().empty() || getElseGraph().empty())
```

Comment:
I think the table gen definition `SizedRegion<1>` implies these are non-empty, could we remove this check or exercise it in a test case if it's needed?

Reply:
Good catch.